app/src/app.js

```diff
@@ -10,7 +10,7 @@
  */
 
 import * as React from 'react';
-import MapScreen from './screens/mapscreen';
+import MapScreen from './screens/MapScreen';
```

Comment:
I'm guessing you renamed the file... I don't know if that works because it still shows as lowercase.

I recall seeing things about how sourcecontrol doesn't track solely case changes in filenames because it can cause problems on some operating systems (e.g. windows)

Reply:
Weird, I'll figure it out.